src/Propel/Runtime/Map/TableMap.php

```diff
@@ -287,7 +287,7 @@ public function getClassName()
     public function getCollectionClassName()
     {
         $collectionClass = $this->getClassName() . 'Collection';
-        if (class_exists($collectionClass)) {
+        if (class_exists($collectionClass) && !is_subclass_of($collectionClass, '\Propel\Runtime\ActiveRecord\ActiveRecordInterface')) {
```

Comment:
```suggestion
        if (class_exists($collectionClass) && !($collectionClass instanceof \Propel\Runtime\ActiveRecord\ActiveRecordInterface)) {
```

more string because there is no string usage